### PR TITLE
feat:[HL-82] List All Meal Categories API 🍕.

### PR DESCRIPTION
**Pull request by ahmednasr8898**
#### Proposed Changes

- api for list of meal categories.

#### Related Issues

N/A.

#### Additional Information

N/A.

#### Screenshot

N/A.

#### Checklist

- [x] I have tested these changes locally.
- [ ] I have added appropriate documentation or updated existing documentation.
- [ ] I have added appropriate test coverage or updated existing test coverage.
- [ ] I have updated the changelog (if applicable).
- [x] I have followed the project's code style and formatting guidelines.
- [x] I have reviewed and adhered to the project's contributing guidelines.

### CODE REVIEW

File: Networking/Sources/Networking/Requests/MealCategoriesRequest.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// MARK: - MealCategoriesResponse
+public struct MealCategoriesResponse: Decodable {
+    let categories: [Category]
+}
+
+// MARK: - Category
+public struct Category: Decodable {
+    let idCategory, strCategory: String
+    let strCategoryThumb: String
+    let strCategoryDescription: String
```
Comment (ahmdmhasn) [inline Networking/Sources/Networking/Requests/MealCategoriesRequest.swift]: Please write descriptive names and use coding keys for decoding.

File: Healthy/Classes/Entities/MealCategories.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+// MARK: - MealCategories
+struct MealCategories {
+    let categories: [MealCategory]
+}
+
+// MARK: - MealCategory
+struct MealCategory {
+    let idCategory, strCategory: String
+    let strCategoryThumb: String
+    let strCategoryDescription: String
```
Comment (ahmdmhasn) [inline Healthy/Classes/Entities/MealCategories.swift]: Please write descriptive names and use coding keys for decoding.

File: Healthy/Classes/Entities/MealCategories.swift [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+// MARK: - MealCategories
+struct MealCategories {
+    let categories: [MealCategory]
+}
+
+// MARK: - MealCategory
```
Comment (ahmdmhasn) [inline Healthy/Classes/Entities/MealCategories.swift]:
```suggestion
// MARK: - MealCategory
```

**Review by ahmdmhasn — APPROVED** [review]

**Comment by ahmdmhasn** [comment]
Please fix the build failure and reach the reviewers for review.

**Review by Ahattalla — APPROVED** [review]
Well done 🚀🔥

**Pull request merged**